### ISSUE TITLE
50

### ISSUE THREAD

**Issue by johnnieng** [johnnieng/DP-100#1]
QUESTION 50
You are building a recurrent neural network to perform a binary classification.
The training loss, validation loss, training accuracy, and validation accuracy of each training epoch has been provided.
You need to identify whether the classification model is overfitted.
Which of the following is correct?
A. The training loss stays constant and the validation loss stays on a constant value and close to the
training loss value when training the model.
B. The training loss decreases while the validation loss increases when training the model.
C. The training loss stays constant and the validation loss decreases when training the model.
D. The training loss increases while the validation loss decreases when training the model.

**Comment by johnnieng** [comment]
B
